.gitignore

```diff
@@ -1,12 +1,8 @@
-.idea
-.DS_Store
-build
-phpunit.xml
-Resources/doc/_build/*
-nbproject
-coverage
+/build
+/Resources/doc/_build
+/vendor
+/node_modules
+/.sass-cache
 composer.lock
-vendor
-node_modules
-.sass-cache
+phpunit.xml
 .php_cs.cache
```

Comment:
I'll use that order like on all the others projects :

```
/build
/Resources/doc/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
```

Do you want that I let the order folder / files or I put the global ignore at the beginning and the project specific after that ?

Example :

```
/build
/Resources/doc/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml

/node_modules
/.sass-cache
```